dali/pipeline/operators/displacement/displacement_filter_impl_cpu.h

```diff
@@ -110,6 +112,66 @@ class DisplacementFilter<CPUBackend, Displacement,
   USE_OPERATOR_MEMBERS();
 
  private:
+  // TODO(klecki) We could probably interpolate with something other than float,
+  // for example integer type
+  struct LinearCoefs {
+    LinearCoefs(float rx, float ry) : rx(rx), ry(ry), mrx(1.f - rx), mry(1.f - ry) {}
+    LinearCoefs() = default;
```

Comment:
It is unclear to me what is the benefit of having a default constructor as values seem to be know on initialization. In that case, initializing it in an inconsistent state and filling the members one by one seems an error-prone approach. I get the idea of a POD, but here you can use static typing to ensure correctness

Reply:
Done.